src/types/utils.cpp

```diff
@@ -1172,7 +1174,10 @@ til::color Utils::ColorFromHexString(const std::string_view str)
 //      in the table, throws runtime_error
 til::color Utils::ColorFromXOrgAppColorName(const std::wstring_view wstr)
 {
-    const std::wstring key(wstr);
+    std::string key = ConvertToA(CP_UTF8, wstr);
+    std::transform(std::begin(key), std::end(key), std::begin(key), [](const std::string::value_type& x) {

```

Comment:
If you use `wstring_view` literals in the map (i.e. `L"whatever"sv`) you wouldn't need to convert the key to utf8 here. Also note that you can simplify that lambda parameter a little by making it `const auto`.